.github/workflows/tests.yml

```diff
@@ -60,7 +60,7 @@ jobs:
 
       # build sphinx
       - name: sphinx
-        run: if [ "${{ matrix.plone-version }}" == "6.0" ] && [ ${{ matrix.python-version }} == '3.12' ]; then make docs-html; fi
+        run: if [ "${{ matrix.plone-version }}" == "6.0" ] && [ ${{ matrix.python-version }} == '3.12' ]; then pip install -r requirements-docs.txt && make docs-html; fi
```

Comment:
@stevepiercy If I'm reading the Makefile correctly, `make docs-html` already depends on `bin/python`, which runs this pip install. So it shouldn't be necessary to do it explicitly here.

Reply:
Good catch. Thank you! Supplemental PR https://github.com/plone/plone.restapi/pull/1817